tests/manifest_v3_test.yaml

```diff
@@ -8,7 +8,9 @@ model:
 # object type definitions
 types:
   #  user represents a user that can be granted role(s)
-  user: {}
+  user:
```

Comment:
Do we need this change? It was intentional to have a NIL object in there?

Reply:
The relation tests create `user --manager--> user` relations, which aren't defined in the manifest.

Reply:
Got it!